openfisca_core/scripts/run_test.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import sys
+import os
+import importlib
+
+from openfisca_core.tools.test_runner import run_tests
+from openfisca_core.tools import detect_country_packages
+
+
+def build_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('path', help = "paths (files or directories) of tests to execute", nargs = '+')
+    parser.add_argument('-c', '--country_package', action = 'store', help = 'country package to use to run the test. If not provided, an automatic detection will be atempted.')
```

Comment:
Much better, yes.

> Should that be part of the help text ?

It should at least be accessible in the man / doc. The easiest way to make it accessible is probably to put it in the help text, even though I agree it's a bit low-level information. Basically, I don't want the user to blindly try options “just in case it could solve the problem”.